postgres/datadog_checks/postgres/postgres.py

```diff
@@ -113,6 +114,7 @@ def dynamic_queries(self):
 
         if self.version >= V10:
             queries.append(QUERY_PG_STAT_WAL_RECEIVER)
+            queries.append(QUERY_PG_REPLICATION_SLOTS)
```

Comment:
should this be gated by some config or is it applicable to standalone postgres instances too?

Reply:
Standalone instances or cluster that are not using replication slots will return 0 rows

```
postgres=# select * from pg_replication_slots ;
 slot_name | plugin | slot_type | datoid | database | temporary | active | active_pid | xmin | catalog_xmin | restart_lsn | confirmed_flush_lsn | wal_status | safe_wal_size | two_phase
-----------+--------+-----------+--------+----------+-----------+--------+------------+------+--------------+-------------+---------------------+------------+---------------+-----------
(0 rows)
```

Given that replication slots are easy to overlook, I would be partial to have this by default.
If there's no replication_slots, no metrics are generated and if there are, the overhead of the function is [super low](https://github.com/postgres/postgres/blob/d0460a31de6acc8bbb86e9e3d646f1113ebb0c20/src/backend/replication/slotfuncs.c#L233) and the number is limited (10 maximum [by default](https://postgresqlco.nf/doc/en/param/max_replication_slots/))

Reply:
OK, that makes sense. Thanks!